sc/social/like/browser/canonicalurl.py

```diff
@@ -42,11 +30,12 @@ class ICanonicalURLUpdater(model.Schema):
         constraint=validate_canonical_domain,
     )
 
-    created_before = schema.Date(
+    published_before = schema.Date(
```

Comment:
I miss the 'date' name in this attribute, how about to change it to 'published_before_date'?